### PR TITLE
bump 4.3.13

### PR DESCRIPTION
**Pull request by mjudeikis**

### CODE REVIEW

File: pkg/util/version/version.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ package version
 // Licensed under the Apache License 2.0.
 
 const (
-	OpenShiftVersion    = "4.3.10"
-	OpenShiftPullSpec   = "quay.io/openshift-release-dev/ocp-release@sha256:edb4364367cff4f751ffdc032bc830a469548f998127b523047a8dd518c472cd"
-	OpenShiftMustGather = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7b4b7938d33cc9f86d822cf6a4dbffb99444566ef93f45a313f600cad11cfee3"
+	OpenShiftVersion    = "4.3.12"
+	OpenShiftPullSpec   = "quay.io/openshift-release-dev/ocp-release@sha256:75e8f20e9d5a8fcf5bba4b8f7d17057463e222e350bcfc3cf7ea2c47f7d8ba5d"
+	OpenShiftMustGather = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a37e5bef81b80c86ab3864c3395d69c0867ab0aa58e1150eceb85be8951def49"
```
Comment (jim-minter) [inline pkg/util/version/version.go:9]: where did you get that ID from? shouldn't it be `dec10f73956f9db4e4e6c878c1dcacba0f17ebe4146c2032d2005f57ad120d0c`?

Comment (mjudeikis) [reply pkg/util/version/version.go]: I think I misscopied from `oc adm release info`. we might need to automate it somehow

**Comment by jim-minter** [comment]
let's go straight to 4.3.13 please

**Comment by mjudeikis** [comment]
> let's go straight to 4.3.13 please

Will wait until 13 lands in stable.

```
[mjudeiki@unknown ARO-RP]$ curl -s -H "Accept: application/json" https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.3 | jq '.nodes[] | select(.version == "4.3.13")'
[mjudeiki@unknown ARO-RP]$ curl -s -H "Accept: application/json" https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.3 | jq '.nodes[] | select(.version == "4.3.12")'
{
  "version": "4.3.12",
  "payload": "quay.io/openshift-release-dev/ocp-release@sha256:75e8f20e9d5a8fcf5bba4b8f7d17057463e222e350bcfc3cf7ea2c47f7d8ba5d",
  "metadata": {
    "description": "",
    "io.openshift.upgrades.graph.release.channels": "candidate-4.3,fast-4.3,stable-4.3,candidate-4.4,fast-4.4,stable-4.4",
    "io.openshift.upgrades.graph.release.manifestref": "sha256:75e8f20e9d5a8fcf5bba4b8f7d17057463e222e350bcfc3cf7ea2c47f7d8ba5d",
    "url": "https://access.redhat.com/errata/RHBA-2020:1393"
  }
}
```

**Comment by jim-minter** [comment]
Reopening.  See #521.  Please prepare the PR, we won't merge it until the e2e passes.

**Comment by mjudeikis** [comment]
```
(pyenv) [mjudeiki@unknown ARO-RP]$ curl -s -H "Accept: application/json" https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.3 | jq '.nodes[] | select(.version == "4.3.13")'
{
  "version": "4.3.13",
  "payload": "quay.io/openshift-release-dev/ocp-release@sha256:e1ebc7295248a8394afb8d8d918060a7cc3de12c491283b317b80b26deedfe61",
  "metadata": {
    "description": "",
    "io.openshift.upgrades.graph.release.channels": "candidate-4.3,fast-4.3,stable-4.3,candidate-4.4,fast-4.4,stable-4.4",
    "io.openshift.upgrades.graph.release.manifestref": "sha256:e1ebc7295248a8394afb8d8d918060a7cc3de12c491283b317b80b26deedfe61",
    "url": "https://access.redhat.com/errata/RHBA-2020:1482"
  }
}
(pyenv) [mjudeiki@unknown ARO-RP]$ oc adm release info quay.io/openshift-release-dev/ocp-release@sha256:e1ebc7295248a8394afb8d8d918060a7cc3de12c491283b317b80b26deedfe61 | grep must
  must-gather                                   sha256:a37e5bef81b80c86ab3864c3395d69c0867ab0aa58e1150eceb85be8951def49
```

**Comment by jim-minter** [comment]
> Now in stable

FYI mirroring changed so that we should now mirror in advance of being in stable

**Comment by mjudeikis** [comment]
our visibility of "What is synced is very limited". I think this is a gap for `mirror`

**Comment by jim-minter** [comment]
flake https://github.com/Azure/ARO-RP/issues/541

/azp run e2e

**Comment by jim-minter** [comment]
we now mirror everything of the form 4.3.\d+ from https://openshift-release.svc.ci.openshift.org/.  You can see the pipeline logs in VSTS.

**Comment by mjudeikis** [comment]
> we now mirror everything of the form 4.3.\d+ from https://openshift-release.svc.ci.openshift.org/. You can see the pipeline logs in VSTS.

I see logs, but I can't see tags manifests in Azure console because we are not transferring tags
https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags

**Pull request merged**